app/controllers/tags_controller.rb

```diff
@@ -104,6 +105,25 @@ def build_editor
     use_javascript('tags/edit')
   end
 
+  def og_data
+    data = {
+      url: tag_url(@tag),
+      title: "#{@tag.name} · #{@tag.type.titleize}",
+    }
+    desc = []
+    desc << generate_short(@tag.description) if @tag.description.present?
+    stats = []
+    post_count = @tag.posts.count
+    stats << "#{post_count} " + "post".pluralize(post_count) if post_count > 0
+    gallery_count = @tag.galleries.count
+    stats << "#{gallery_count} " + "gallery".pluralize(gallery_count) if gallery_count > 0
+    character_count = @tag.characters.count
+    stats << "#{character_count} " + "character".pluralize(character_count) if character_count > 0
+    desc << stats.join(', ')
+    data[:description] = desc.join("\n")
```

Comment:
If the tag is a setting, perhaps mention who it's owned by (if it's, in fact, owned)?

Reply:
Do you have a suggestion on where to put that?

Reply:
Hmm. I would _probably_ choose to put it between the tag name and the tag type (so, like, `Proximal · Throne3d · Setting`), to match the fact we have a similar thing on posts (`Between and beyond stations · Kel, Throne3d`), but I'm not sure if that's the best place. Maybe put it there and then we ask for feedback on it later?